azure-pipelines.yml

```diff
@@ -874,4 +906,19 @@ jobs:
     inputs:
       PathtoPublish: $(Build.ArtifactStagingDirectory)/zip
       ArtifactName: zipv2
+
+  - task: SdtReport@2
+    inputs:
+      GdnExportAllTools: true
+      GdnExportHtmlFile: true
```

Comment:
Small Suggestion/Idea: Currently report is exported only for errors , do we want to add "warnings" as well? because warnings turn into errors sometimes. :)
```GdnExportGdnToolCredScanSeverity: Warning```

Reply:
Will it also fail on warning? We only want the pipeline to fail on error right?

Reply:
No it will export both Errors and Warnings in the report. It will break only on Error or Errors and Warnings which can be configured via PostAnalysis step 
```- task: PostAnalysis@2
  displayName: 'Post Analysis'
  inputs:
    GdnBreakPolicyMinSev: Error``` Values can be Error or Warnings here. It will break the build based on this value.

Reply:
tested this flags, it doesnt change what shows up on logs of post analysis. Keeping it as error for now.